app/lib/Config.scala

```diff
@@ -2,17 +2,26 @@ package lib
 
 import java.io.FileNotFoundException
 import java.net.URI
-import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
+
 import scala.io.Source
+import scala.util.Try
 
 object Config {
-  val properties = Properties.fromPath("/etc/gu/s3-uploader.properties")
+  val properties: Map[String, String] = Try(Properties.fromPath("/etc/gu/s3-uploader.properties")).getOrElse(Map.empty)
+
+  val awsCredentials = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("media-service"),
+    InstanceProfileCredentialsProvider.getInstance()
+  )
 
-  val awsCredentials = new DefaultAWSCredentialsProviderChain()
+  val region = properties.getOrElse("aws.region", "eu-west-1")
```

Comment:
Presumably this means its zero-config in DEV?

Reply:
Yeah it felt like the most straight-forward thing to do